src/schematics/src/update/update-0.13.0.ts

```diff
@@ -76,6 +76,17 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Change non-Clr prefixed classes to include the Clr prefix.
+          // We are using Cl instead of Clr in the regex for simplicity, because we export
+          // `ClarityModule,` which would be covered by this regex.
+          const reg = /import\s*{([^}]*)}\s*from\s*['"]@clr\/angular['"]/g;
+          let result = reg.exec(updated);
+          while (result !== null) {
+            const newString = result[1].replace(/\b(?!Cl)\w+\b/g, 'Clr$&');
+            updated = updated.replace(result[1], newString);
```

Comment:
Yes, as @youdz says we're not capturing all instances from the file. What we should be doing is for any match in the import that is missing `Clr`, we should search for it and replace it in the whole file.